resources/contenttweaker/blockstates/amuileria_kaerunea.json

```diff
@@ -0,0 +1,18 @@
+{
+  "forge_marker": 1,
+  "variants": {
+    "normal": [{
+      "model": "botania:shapes/cross_tinted",
+      "textures": {
+        "cross": "contenttweaker:blocks/amuileria_kaerunea"
+      }
+    }],
+    "inventory": [{
+      "model": "builtin/generated",
+      "transform": "forge:default-item",
+      "textures": {
+        "layer0": "contenttweaker:blocks/amuileria_kaerunea"
+      }
+    }]
+  }
+}
```

Comment:
If this blockstates automatically generated, they should be [ignored by repo](https://github.com/Krutoy242/Enigmatica2Expert-Extended/blob/aadb1a512491d2ee2abc4c0436b5fe32407393ed/.gitignore#L80). Please remove them from PR if so.

For the future, if you want to add non-default blockstate, use `git add -f` for changed states.